client/asset/dcr/dcr.go

```diff
@@ -1596,13 +1596,76 @@ func (dcr *ExchangeWallet) AuditContract(coinID, contract, txData dex.Bytes) (*a
 	if txOut == nil {
 		return nil, asset.CoinNotFoundError
 	}
+
+	// Since txOut.Version with dcrd 1.6 is the transaction version NOT the
+	// output script version, we need to pull the entire transaction to get it.
+	// Unfortunately, for confirmed transactions we cannot do that with
+	// getrawtransaction without txindex. If confirmed, use
+	// GetTxOutResult.Confirmations and BestBlock to pull the block that is
+	// expected to include this tx.
+	//
+	// In dcrd 1.7, the actual script version is in
+	// GetTxOutResult.ScriptPubKey.Version, and GetTxOutResult.Version is
+	// removed:
+	//
+	// const scriptversion uint16 = txOut.ScriptPubKey.Version // dcrd/dcrwallet 1.7 *release*, not yet
+	var verboseTx *chainjson.TxRawResult
+	if txOut.Confirmations > 0 {
+		bestBlockHash, err := chainhash.NewHashFromStr(txOut.BestBlock)
+		if err != nil {
+			return nil, fmt.Errorf("error decoding best block hash %s: %w",
+				txOut.BestBlock, err)
+		}
+		block, err := dcr.node.GetBlockVerbose(dcr.ctx, bestBlockHash, true)
+		if err != nil {
+			return nil, fmt.Errorf("error getting block header: %w", translateRPCCancelErr(err))
+		}
+		if txOut.Confirmations > 0 { // 1 means it is in the best block above
```

Comment:
```Go
if txOut.Confirmations > 1 {
```
?

Reply:
Yes!  I hacked it like this temporarily to test this, but didn't change back